### PR TITLE
Folder structure change

### PR DESCRIPTION
**Pull request by parulrawat-compro**

### CODE REVIEW

File: Client/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
     "bootstrap": "^4.3.1",
     "jquery": "^3.4.1"
+    "node-sass": "^4.5.3",
```
Comment (manpreet-compro) [inline Client/package.json]: why this is required?

File: Client/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+import { AppComponent } from './app.component'
 
 
 const routes: Routes = [
-  { path: '', redirectTo: '/login', 'pathMatch': 'full' },
-  { path: 'pagenotfound', component: PageNotFoundComponent },
-  { path: 'login', component: LoginPageComponent },
-  { path: 'dashboard', component: DashboardComponent },
+  {
+    path: 'app',
```
Comment (manpreet-compro) [inline Client/src/app/app-routing.module.ts]: Login should be at route level, not within app

File: Client/package.json (hunk 1 above)
Comment (parulrawat-compro) [reply Client/package.json]: Agreed, also this is not required as such hence removing

File: Client/src/app/app-routing.module.ts (hunk 2 above)
Comment (parulrawat-compro) [reply Client/src/app/app-routing.module.ts]: Updated

**Pull request merged**